src/custom/pages/Claim/types.ts

```diff
@@ -7,3 +8,11 @@ export type ClaimCommonTypes = {
   tokenCurrencyAmount: CurrencyAmount<Token>
   handleChangeAccount: (e: SyntheticEvent<HTMLButtonElement>) => void
 }
+
+export type ParsedUserClaim = UserClaimData & {
```

Comment:
Can we add some coments to explain what is the goal of this `ParsedUserClaim`.

Is this just an object with some augmented info for the claiming object? (claiming info from the chunks + price + currency + etc)
if so, can we just call it `UserClaimDataDetails` for example?

Reply:
Yep, done